### PR TITLE
Feature/homepage

### PR DESCRIPTION
**Pull request by JotGabiga**

### CODE REVIEW

**Review by mateugazi — APPROVED** [review]

File: src/app/HomePage.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
+'use strict';
+import { MainMenu } from './MainMenu.js';
```
Comment (mateugazi) [inline src/app/HomePage.js:2]: You should import menu from GlobalData not from the MainMenu

**Review by mateugazi — CHANGES_REQUESTED** [review]
You should import menu from GlobalData not from the MainMenu

File: src/app/HomePage.js [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+const URLSearchByIngredients = 'https://api.spoonacular.com/recipes/findByIngredients';
+const URLRandomRecipes = 'https://api.spoonacular.com/recipes/random';
+
+const tagsElement = document.getElementById("tags");
+const recipeSection = document.getElementById("recipeSection");
+const searchButton = document.getElementById("searchButton");
+const ingredientsInput = document.getElementById("ingredient");
+const tags = document.getElementById("tags");
+
+MainMenu();
```
Comment (kingam280) [inline src/app/HomePage.js:14]: You need to add activePage argument here, it requires data from GlobalData

File: index.html [hunk 3]
```diff
@@ -9,17 +9,37 @@
...
+        <div class="searchDescription">
+            Look for a meal by the ingredients you already have
+        </div>
+        <div class="searchSection">
+            <div id="tags"></div>
+            <div class="inputWrapper">
+                <form>
+                    <input id="ingredient" type="text" placeholder="e.g. pasta, tuna" name="search">
+                </form>
+                <button id="searchButton"><i class="fa fa-search"></i></button>
```
Comment (kingam280) [inline index.html:31]: Move button into the form. It will work not only when button clicked but also on enter

File: src/app/HomePage.js [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+export function createElementWithClass(element, ...elementClass) {
+    let newElement = document.createElement(element);
+    newElement.classList.add(...elementClass);
+    return newElement;
+}
+
+window.addEventListener('load', function() {
+    getRandomRecipes();
+    searchButton.addEventListener("click", function() {
+        getRecipesByIngredient(getInputValue())
```
Comment (kingam280) [inline src/app/HomePage.js:147]: After moving button into form, here event.preventDefault() needs to be inserted

**Review by kingam280 — CHANGES_REQUESTED** [review]

File: src/app/HomePage.js (hunk 4 above)
Comment (JotGabiga) [reply src/app/HomePage.js:147]: Thanks for that tip, I had problem with this button.

**Review by kingam280 — DISMISSED** [review]

**Review by Yoruko90 — DISMISSED** [review]

**Review by mateugazi — APPROVED** [review]

**Review by kingam280 — DISMISSED** [review]

**Review by kingam280 — APPROVED** [review]

**Review by mateugazi — APPROVED** [review]

**Pull request merged**